website/docs/r/cloud_project_database.html.markdown

```diff
@@ -94,6 +94,10 @@ resource "ovh_cloud_project_database" "mysqldb" {
     region  = "SBG"
   }
   flavor        = "db1-4"
+  advanced_configuration = {
+    "mysql.sql_mode": "ANSI,ERROR_FOR_DIVISION_BY_ZERO,NO_ENGINE_SUBSTITUTION,NO_ZERO_DATE,NO_ZERO_IN_DATE,STRICT_ALL_TABLES",
```

Comment:
What about the available advanced configuration for other engine than mysql? :)

Reply:
There is too much advanced configuration, especially if we add up all the engines. 
Each engine already has a page for advanced parameters in the public documentation.
Here is just an example to how use it and can be adapted to all other advanced parameters